strings/ion_laws.json

```diff
@@ -837,7 +835,8 @@
         "SLIME PEOPLE",
         "GOLEMS",
         "SHADOW PEOPLE",
-        "CHANGELINGS"
+        "CHANGELINGS",
+        "MOTHMEN"
```

Comment:
erm... you meant to say "MONTHMEN"?

Reply:
No, no no I didnt make this, but the person that did was trying to remove primarily this

Reply:
![image](https://user-images.githubusercontent.com/60765218/123177097-a4aadd00-d452-11eb-962a-f053b1e3b992.png)

Reply:
![image](https://user-images.githubusercontent.com/60765218/123177109-aaa0be00-d452-11eb-99fb-17b70e35d7f2.png)